File: 競プロ典型 90 問/004_Cross_Sum.py
# -*- coding: utf-8 -*-
import numpy as np
h, w = map(int, input().split())

array = []
yoko = []
tate = np.zeros(w)


for i in range(h):
    tmp_array = list(map(int, input().split()))
    yoko.append(sum(tmp_array))
    tate += tmp_array
    array.append(tmp_array)
    

for i in range(h):
    for j in range(w):
        print(int(tate[j]+yoko[i]-array[i][j]), end=" ")
    print()

File: 競プロ典型 90 問/004_Cross_Sum.cc
#include <iostream>

int main(){
 int h, w, i, j, tmp, l;
  std::cin >> h >> w;
  int a[h][w], yoko[h]={}, tate[w]={};
  for(i=0; i<h; i++){
    for(j=0; j<w; j++){
      std::cin >> a[i][j];
      tate[j] += a[i][j];
      yoko[i] += a[i][j];
    }
  }

  for(i=0; i<h; i++){
    for(j=0; j<w; j++){
      std::cout << tate[j]+yoko[i]-a[i][j] << ' ';
    }
    std::cout << '\n';
  }

  return 0;
}
